NUAttribute.js

```diff
@@ -15,7 +15,7 @@ export default class NUAttribute extends NUObject {
     static ATTR_TYPE_LONG = 'long';
     static ATTR_TYPE_STRING = 'string';
     static ATTR_TYPE_OBJECT = 'object';
-
+    static ATTR_TYPE_DATE = 'long';
```

Comment:
Can we actually call this then ATTR_TYPE_TIMESTAMP? Just in case we want in the future to use a Date attribute as in Date object - not sure if it will happen but TIMESTAMP is more appropriate in any case. And please add this in the validation function